### ISSUE TITLE
NVDA is announcing the list item name ‘No’ as ‘Number’ which is present at overwrite existing files dropdown.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13267]
https://user-images.githubusercontent.com/93735775/150609122-dce4eec2-6c02-4ff3-96f8-46fa8545cba2.mp4

### Steps to reproduce:
1.	Turn on screen reader "NVDA".
2.	Hit the link: [A Pen by Leaundrae (codepen.io)](https://codepen.io/lmckinney/pen/dyzwgBG?editors=1010)
3.	Tab till overwrite existing files dropdown and press enter to expand it.
4.	Use up/down arrow key to navigate to ‘No’ option and verify NVDA is announcing list item name ‘No’ as ‘Number’ or not

### Actual behavior:
After expanding the overwrite existing files dropdown, when keyboard focus comes to ‘No’ option, NVDA is announcing it as ‘Number’. NVDA does this in default verbosity.

### Expected behavior:
NVDA should announce the list item name properly. It should announce the name of the list item as ‘No’ not as ‘Number’

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2021.2

#### Windows version:
Windows 10 Version 21H2 (OS Build 22000.318)

#### Name and version of other software in use when reproducing the issue:
Google Chrome Version 97.0.4692.71 (Official Build) (64-bit)
#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
2021.3.1

**Comment by feerrenrut** [comment]
This is the synthesizer (Windows One Core voices) behavior. Note that the speech viewer, shows "No" not "Number". After swapping to espeak-ng, this is spoken as "No".

Please raise this issue with the Windows One Core team. Closing as 'can't fix', as it is internal.
